app/assets/stylesheets/application.css.scss

```diff
@@ -273,3 +273,7 @@ a:hover {
 	color:#800000;
 	border: 2px dashed #EEAA88;
 }
+
+.forttree-banner {
+    max-height: 400px;
```

Comment:
@houndci Umm no I prefer 4 spaces. Besides, the css clause above this is indented with a tab, you should be complaining about that.